platform.json

```diff
@@ -59,7 +59,6 @@
     },
     "tool-openocd": {
       "type": "uploader",
-      "optional": true,
```

Comment:
Just curious, why the OpenOCD package is no longer optional?

Reply:
I did this because I feared that tool-openocd wouldn't be installed if it is missing when starting a debug session. Is this not the case? (The GDB server is opened by openocd)

Reply:
Thanks, I think it's safe to return the `optional` field as it will be installed before starting a debug session. As you know we don't always have a complete set of packages for all OSes and if we specify OpenOCD as an optional package then users will be able to at least install the platform and use basic functionality.

Reply:
You are right, I just re-inserted the `optional` attribute, deletec my `tool-openocd` folder and then it was redownloaded at the start of the debug session. Fixed in latest commit.